scss/navbar.scss

```
@@ -1,27 +1,27 @@
 $splash-light-purple: #7c408a;
 
-.navbar {
+.navbar:not(.md-layout-nav) {
   position: absolute !important;
   width: 100%;
   z-index: 100;
   padding-top: 1.25em !important;
   top: 0;
-}
 
-.navbar-nav {
-  font-size: 1.25em !important;
-}
+  .navbar-nav {
+    font-size: 1.25em !important;
```

```suggestion
    font-size: 1.25em !important;
    padding-right: 40px;
```

This is necessary to make the nav perfectly center